docs/source/enterprise/index.rst

```diff
@@ -165,11 +165,11 @@ pages on this site apply to Teams deployments as well.
     Cloud-backed media <cloud_media>
     Roles and permissions <roles_and_permissions>
     Dataset Versioning <dataset_versioning>
-    FiftyOne Teams App <teams_app>
+    FiftyOne Enterprise App <enterprise_app>
     Data Lens __SUB_NEW__ <data_lens>
     Data Quality __SUB_NEW__ <data_quality>
     Query Performance __SUB_NEW__ <query_performance>
-    Plugins <teams_plugins>
+    Plugins <enterprise_plugins>
```

Comment:
nit: it was unnecessary and inconsistent for us to previously call this page `teams_plugins`. Now that we're renaming, how about just `plugins` here? For consistency with the other page titles here.

Reply:
sure